cmdrs-guide.html

```diff
@@ -0,0 +1,137 @@
+            greatly enhances the ease of playing Elite Dangerous. <b>These tools are not the official tools of Elite
+            Dangerous
+            and their developers have no affiliation to Frontier Developments.</b>
+        </p>
+        <p>
+            Please keep in mind that these tools are created and maintained by developers for free in their free time at
+            a best effort basis. Some of these developers can be supported via various channels. If possible, please
+            support their efforts. Links would be available in the respective locations of the tools.
+        </p>
+        <h3>What are these tools and where can I find them?</h3>
+        <p>
+            You can find links to some of these tools in the <a href="index.html">homepage</a>. You can find more at
+            <a href="http://edcodex.info/">Elite:Dangerous Codex</a>. There are a vast number of tools, big or small
+            that you might discover but these should be enough to get you started.
+        </p>
+        <h3>What am I supposed to do with these tools?</h3>
+        <p>
+            Like all great questions, the answer is; it depends. There are broadly 2 types of tools:
+        </p>
+        <ol>
+            <li>
+                Client tools, running on your PC.
+            </li>
+            <li>
+                Browser based tools, running on some remote server and accessed using your web browser.
+            </li>
+        </ol>
+        <p>
+            Client tools would need to be downloaded and run from your system. Most of the tools need to be run on the
+            same system that Elite Dangerous is running on although some tools might not need so.
+        </p>
+        <p>
+            The available tools can be used for anything from tracking and logging your activities, keeping track of
+            your resources and credits, to giving advice for how to build that perfect ship you are looking for.
+        </p>
+        <h3>How do these tools get their data?</h3>
+        <p>
+            There are 3 ways tools get their data:
+        </p>
+        <ol>
+            <li>
+                From journal files generated by Elite Dangerous. Some use the data of these files sent through EDDN.
+            </li>
+            <li>
+                From CAPI once you authenticate your Frontier account with the tools.
+            </li>
+            <li>
+                From manual user input
+            </li>
+        </ol>
+        <h3>Whoa! What are journal files, EDDN and CAPI?</h3>
+        <h4>Journal Files</h4>
+        <p>
+            When you start Elite Dangerous and do activities in it, the game writes some particular files in the system
+            you are playing on. These files keep track of your activities by writing
+            <a href="https://www.w3schools.com/whatis/whatis_json.asp">JSON</a> data. These files are called the Journal
```

Comment:
```suggestion
            <a href="https://en.wikipedia.org/wiki/JSON">JSON</a> data. These files are called the Journal
```

Reply:
I used the w3school link since it explains things from a learners perspective.